src/pudl/transform/eia861.py

```diff
@@ -731,6 +731,72 @@ def advanced_metering_infrastructure(tfr_dfs):
         dict: A dictionary of transformed EIA 861 dataframes, keyed by table name.
 
     """
+    idx_cols = [
+        "utility_id_eia",
+        "state",
+        "balancing_authority_code_eia",
+        "report_date",
+    ]
+
+    ###########################################################################
+    # Tidy Data:
+    ###########################################################################
+    logger.info("Tidying the EIA 861 Advanced Metering Infrastructure table.")
+    # Clean up values just enough to use primary key columns as a multi-index:
+    # Transform values with fix_eia_na()
+    logger.debug(
+        "Cleaning up EIA861 Advanced Metering Infrastructure index columns so we can tidy data.")
+    clean_ami = (
+        tfr_dfs["advanced_metering_infrastructure_eia861"].copy()
+        .assign(balancing_authority_code_eia=lambda x: x.balancing_authority_code_eia.fillna("UNK"))
+        .dropna(subset=["utility_id_eia"])
+        .query("utility_id_eia not in (88888, 99999)")
```

Comment:
Just checking, were these operations actually required for this table? Or is it just cut-and-paste from the `sales_eia861` table?

Reply:
These were just cut and pasted from sales.

Reply:
I think we should probably remove any operations that aren't actually required.